src/redis.js

```diff
@@ -0,0 +1,39 @@
+const logger = require('./logger');
+const Redis = require('ioredis');
+const MockRedis = require('ioredis-mock');
+
+// If you need to set the Redis URL, do it in REDIS_URL
+const redisUrl = 'http://redis:6379';
```

Comment:
This seems wrong.  First, it's using `http://` vs. `redis://`, and second, it's not using an env variable.